src/main/frontend/src/routes/_layout.svelte

```diff
@@ -1,17 +1,27 @@
 <script>
 	import { onMount } from 'svelte';
-	import { theme, isMobile } from '../stores';
+	import { theme, isMobile, settingsInfo } from '../stores';
 	import { Button, Icon, MaterialApp, AppBar, Container } from "svelte-materialify/src";
 	import { mdiMenu, mdiWeatherNight, mdiWeatherSunny } from '@mdi/js';
 	import SiteNavigation from '../components/SiteNavigation.svelte';
 	export let segment;
+	import Repository from '../api/Repository';
 
 	let sidenav = false;
 	const toggleTheme = () => $theme = ($theme === "light") ? "dark" : "light";
 	$: bgTheme = ($theme === "light") ? "#ffffff" : "#212121";
 
 	onMount(() => {
 		isMobile.check();
+		Repository.get('/generation-settings/info')
+			.then(response => response.json())
+			.then(data => {
+				if ($settingsInfo.xoomDesignerFileVersion !== data.xoomDesignerFileVersion) {
+					localStorage.clear();
+				} else {
+					$settingsInfo = data;					
```

Comment:
Shouldn't the `$settingsInfo` always be set to `data` even when `localStorage` is cleared? In other words, should the else be removed and the `$settingsInfo = data;` always happen?

Reply:
@VaughnVernon yes you're right, it should be set in both cases